### PR TITLE
Adapt editor for intersecting order of parent/job groups

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

**Comment by Martchus** [comment]
@okurz So you are fine with the use of 'Add folder' to distinguish between adding top-level job group and folder/parent?

**Comment by okurz** [comment]
please give me a screenshot

**Comment by okurz** [comment]
hm, checked it locally, not that convincing. Let me visit you in person.

File: templates/admin/group/group_row.html.ep [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<li
+    id="<%= $is_parent ? 'parent' : 'job' %>_group_<%= $group->id %>"
```
Comment (coolo) [inline templates/admin/group/group_row.html.ep]: you have so many is_parents in here that I almost believe having 2 templates is easier to read - the duplicated code seems rather limited

Comment (Martchus) [inline templates/admin/group/group_row.html.ep]: This is correct. I'll spit it again.

**Comment by coolo** [comment]
LGTM

**Comment by okurz** [comment]
the folder icon now looks a bit like a first aid kit but I don't mind :-D

**Comment by coolo** [comment]
if just Marius wouldn't push commits all the time, I would have merge :)

**Comment by Martchus** [comment]
I had to push another commit because the tests didn't pass.

**Comment by okurz** [comment]
LGTM

**Comment by coolo** [comment]
btw: my congrats for PR 1000

**Pull request merged**